src/server/compositor/multi_monitor_arbiter.cpp

```diff
@@ -48,8 +48,10 @@ std::shared_ptr<mg::Buffer> mc::MultiMonitorArbiter::compositor_acquire(composit
     if (current_buffer_users.find(id) != current_buffer_users.end() || !current_buffer)
     {
         if (schedule->num_scheduled())
+        {
             current_buffer = schedule->next_buffer();
-        current_buffer_users.clear();
+            current_buffer_users.clear();
+        }
```

Comment:
This isn't just formating.

Reply:
Correct you are. Like I said, this branch was old and these lines slipped my review. I believe the change to be correct, so I've moved the commit to another branch which I will PR separately later.